lib/node_modules/@stdlib/blas/ext/base/gcusumkbn/docs/types/index.d.ts

```diff
@@ -44,7 +54,7 @@ interface Routine {
 	* gcusumkbn( x.length, 0.0, x, 1, y, 1 );
 	* // y => [ 1.0, -1.0, 1.0 ]
 	*/
-	( N: number, sum: number, x: NumericArray, strideX: number, y: NumericArray, strideY: number ): NumericArray;
+	( N: number, sum: number, x: InputArray, strideX: number, y: OutputArray, strideY: number ): OutputArray;
```

Comment:
```suggestion
	<T extends OutputArray>( N: number, sum: number, x: InputArray, strideX: number, y: OutputArray, strideY: number ): T;
```

Allow preserving output array type specificity.